template/lib.typ

```diff
@@ -227,27 +227,151 @@ text)
   }
 
 
+#let pretty-header(numeration: "1", main: false) = {
```

Comment:
I know it's pretty, but that's a given :) Shouldn't we name it `page-header` instead? At first I thought this was for headings instead of page headers